File: api-examples/rs/count/Go/spatial_bounding_box.go
package main

import (
  "fmt"
  "strings"
  "net/http"
  "io/ioutil"
)

func main() {

  url := "https://%3Cresource-server-ip%3E/resource-server/pscdcl/v1/count"
  method := "POST"

  payload := strings.NewReader("{\n	\"id\": \"rbccps.org/aa9d66a000d94a78895de8d4c0b3a67f3450e531/pudx-resource-server/changebhai/crowd-sourced-images\",\n	\"bbox\":\"18.2,73.6,24.2,76.6\"\n}")

  client := &http.Client {
  }
  req, err := http.NewRequest(method, url, payload)

  if err != nil {
    fmt.Println(err)
  }
  req.Header.Add("Content-Type", "application/json")

  res, err := client.Do(req)
  defer res.Body.Close()
  body, err := ioutil.ReadAll(res.Body)

  fmt.Println(string(body))
}

File: api-examples/rs/search/js/spatial_bounding_box.js
var settings = {
  "url": "https://<resource-server-ip>/resource-server/pscdcl/v1/search",
  "method": "POST",
  "timeout": 0,
  "headers": {
    "Content-Type": "application/json"
  },
  "data": "{\n\t\"id\": \"rbccps.org/aa9d66a000d94a78895de8d4c0b3a67f3450e531/pudx-resource-server/changebhai/crowd-sourced-images\",\n\t\"bbox\":\"18.2,73.6,24.2,76.6\"\n}",
};

$.ajax(settings).done(function (response) {
  console.log(response);
});
